Library/.rubocop.yml

```diff
@@ -339,13 +339,47 @@ Style/DisableCopsWithinSourceCodeDirective:
     - "/**/{Formula,Casks}/*.rb"
     - "**/{Formula,Casks}/*.rb"
 
-# Don't enforce documentation in casks or formulae.
+# Only enforce documentation for public APIs.
 Style/Documentation:
-  Exclude:
-    - "Taps/**/*"
-    - "/**/{Formula,Casks}/*.rb"
-    - "**/{Formula,Casks}/*.rb"
-    - "**/*.rbi"
+  AllowedConstants:
+    - Homebrew
+  Include:
+    - Homebrew/cask/dsl.rb
```

Comment:
Feels like this should probably be in the `Homebrew/.rubocop.yml` considering it's scoped entirely to there? Thoughts?